docresources/SyncBot/syncbot-readme.md

```diff
@@ -277,6 +277,43 @@ Errors, where possible, are reported in the originating thread as whispers.
 		<Send to a Webhook> <https://your.deploy/front>
 	```
 
+### Setting up a Discourse category for SyncBot
+
+1) You will need the help of a Discourse admin (not moderator) to perform these steps
+1) Get an All Users API Key from https://your.discourse/admin/api/keys
+1) Fire a request similar to the following to acquire a list of categories
+	```
+	
+	GET https://your.discourse/categories
```

Comment:
Example `curl` line is always handy :)

Reply:
I use this format because it is directly executable using VSCode's Rest Client (which is excellent, by the way) and easier to read.  It is in use throughout this documentation.  If I were to go over to curl here, I would want to go over to curl throughout this file, which would expand the scope of this change beyond necessity.

Reply:
OK, but I'd note that curl examples are by far more prevalent in the technical world than an example for a single IDE (and I disagree on readability, everyone knows how to use curl).